handler.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+const fs = require('fs');
+const { getSurveyResults } = require('./src/get-survey-results');
+const AWS = require('aws-sdk');
+const s3 = new AWS.S3();
+
+module.exports.storeSurvey = async event => {
+  const surveyId = process.env.SURVEY_ID;
+  console.log(`getting data for survey: ${surveyId}`);
+  const fileName = `/tmp/${surveyId}.csv`;
+  const destinationStream = fs.createWriteStream(fileName);
+  await getSurveyResults(
+    process.env.QUALTRICS_API_TOKEN,
+    process.env.QUALTRICS_DATA_CENTER,
+    surveyId,
+    destinationStream
+  );
+
+  return destinationStream.on('finish', async () => {
```

Comment:
@Trott is this the right event to listen to here given that the write is being piped in https://github.com/ucsf-ckm/som-qualtrics-survey-responses/blob/master/src/get-survey-results.js#L52 ? Is there a promisey way to wait for this or is it just an event and that is that?